### ISSUE TITLE
[BUG] `EntryBuffer` trampoline dereferences a null pointer.

### ISSUE THREAD

**Issue by mahkoh** [gtk-rs/gtk4-rs#565]
https://github.com/gtk-rs/gtk4-rs/blob/fb59005041d06bc275de3ff23a740395b6332c1e/gtk4/src/subclass/entry_buffer.rs#L232-L233

`n_bytes` is permitted to be a null pointer and is usually a null pointer.

**Comment by sdroege** [comment]
Indeed, do you want to provide a PR for this? Should simply check for `NULL` before and only assigned the length if it's not `NULL`.
